src/main/java/com/vladbudan/restapp/service/PetService.java

```diff
@@ -0,0 +1,23 @@
+package com.vladbudan.restapp.service;
+
+import com.vladbudan.restapp.model.Cat;
+import com.vladbudan.restapp.model.Dog;
+import com.vladbudan.restapp.model.Pet;
+
+import java.util.List;
+
+public interface PetService {
```

Comment:
Why you didn't create separate services for Cat and Dog as you did with repositories?